### DISCUSSION TITLE
Ridimensionamento video

### DISCUSSION THREAD

**Discussion by Roscopct** [AndreyPavlenko/Fermata#529] [Announcements]
Andrey buonasera.. 
Mi sono accorto che utilizzando youtube in fermata auto e ridimensionando il video per portarlo a full screen.. nel caso di 10 pollici.. con la funzione ritaglia viene distorta l'immagine mantenendo tutto il contenuto e con estendi viene mantenuta la proporzione e viene tagliata parte dell'immagine sopra e sotto... utilizzando tv invece non c'è la funzione estendi e comunque la funzione ritaglia porta il canale in full screen mantenendo la proporzione ma tagliando sopra e sotto l'immagine.. 
A questo punto sarebbe possibile così come in youtube estenderla a full screen senza tagliare il video e distorcendolo solo??
Spero di essere stato chiaro e ringrazio..

**Comment by AndreyPavlenko** [comment]
Good evening.

YouTube is the web application https://m.youtube.com/. Try opening it on your phone and check the possible options to resize the video. Fermata can't change it but only provides controls to switch the options.

**Comment by Roscopct** [comment]
Ok.. e la funzione TV non c è modo di ridimensionare il video a tutto schermo senza tagliare l immagine?? Grazie
